pkgs/by-name/js/json2kdl/package.nix

```diff
@@ -0,0 +1,26 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "json2kdl";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "AgathaSorceress";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-NVpIHbv7vbppe+g7YK9OY2oL7axmqG8Kmuv4kO8Jyjs=";
+  };
+
+  cargoHash = "sha256-xlG8p25VBLwUWnyr9JNzSrI0KmwdRpAgL5eckbC/3nk=";
```

Comment:
it seems this tag has changed since: 

```
error: hash mismatch in fixed-output derivation '/nix/store/k0shsqibh29w4v9mx015dlibgx05f8bj-json2kdl-0.2.0-vendor-staging.drv':
        likely URL: (unknown)
         specified: sha256-xlG8p25VBLwUWnyr9JNzSrI0KmwdRpAgL5eckbC/3nk=
            got:    sha256-PK/DduEy0BfHt0asEUR41lvUl++w/UTqZ0HFSuO2OVI=
```